.github/workflows/ci.yml

```diff
@@ -9,19 +9,19 @@ on:
 jobs:
   pod-lint:
     name: Pod Lint
-    runs-on: macos-13
+    runs-on: macos-14
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
       - name: Bundle Install
         run: bundle install
       - name: Select Xcode Version
-        run: sudo xcode-select --switch /Applications/Xcode_15.0.1.app/Contents/Developer
+        run: sudo xcode-select --switch /Applications/Xcode_15.2.app/Contents/Developer
```

Comment:
Xcode 15.0.1 on macOS 14 is pretty buggy and has a lot of spurious failures. Going to Xcode 15.2 to try to avoid this, plus we need 15.2 for visionOS.